lib/components/DynamicListTile.dart

```diff
@@ -63,7 +93,7 @@ class Fields {
 
```

Comment:
can you move fields into it's own class?

Reply:
Should be able to. I'll look into it